app/main.py

```diff
@@ -1 +1,51 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
Instead of redefining the `self.skills` list defined in the `SoftwareEngineer` class, extend `self.skills` from the `SoftwareEngineer` class with new values so you would not create a new list of skills in each class. Change the `__init__` of other classes that add new skills as well.